compiler/src/iree/compiler/Dialect/Flow/Transforms/test/outline_dispatch_regions.mlir

```diff
@@ -179,11 +179,39 @@ func.func @dispatchWithCountRegion(%arg0: tensor<4xi32>) -> tensor<4xi32> {
 
 // -----
 
+//      CHECK: flow.executable private @main_dispatch_0 {
+// CHECK-NEXT:   flow.executable.export public @main_dispatch_0_generic_4x8_i32xf32
+//      CHECK: func.func @main_dispatch_0_generic_4x8_i32xf32(
```

Comment:
Can you move this test case down further in the file? The `// Dispatches containing some ops get a heuristics-driven summary in their name.` test should come first, then this test case for a specific format of the summary can go after.

(the bottom of the file looks best here)